src/main/java/core/mate/academy/model/BulldozerProducer.java

```diff
@@ -0,0 +1,18 @@
+package core.mate.academy.model;
+
+import core.mate.academy.service.MachineProducer;
+import java.util.List;
+
+public class BulldozerProducer implements MachineProducer<Bulldozer> {
+
+    @Override
+    public List<Bulldozer> get() {
+        List<Bulldozer> listBuldozers = List.of(createBulldozer(),createBulldozer());
+        return listBuldozers;
+    }
+
+    public Bulldozer createBulldozer() {
```

Comment:
let's remove this method it gives us no value you can call constructor in get and have less code

Reply:
done